src/solvers/smt2_incremental/smt_solver_process.cpp

```diff
@@ -28,7 +28,17 @@ void smt_piped_solver_processt::send(const smt_commandt &smt_command)
   const std::string command_string = smt_to_smt2_string(smt_command);
   log.debug() << "Sending command to SMT2 solver - " << command_string
               << messaget::eom;
-  process.send(command_string + "\n");
+  const auto response = process.send(command_string + "\n");
+  switch(response)
+  {
```

Comment:
I seem to recall a mechanism like that existing in the earlier implementation. Did it get removed?

Anyway, nice change 👍🏻